### PR TITLE
[PR] Article font weight

### PR DESCRIPTION
**Pull request by philcable**

### CODE REVIEW

**Comment by philcable** [comment]
@slocker3 #reviewmerge?

File: style.css [hunk 1]
```diff
@@ -681,6 +701,11 @@ h6 {
 	font-size: 18px;
 }
 
+strong,
+b {
+	font-weight: 400;
```
Comment (slocker3) [inline style.css]: @philcable why set `strong` and `b` as the same weight as the paragraph text?

File: css/01-layout.css [hunk 2]
```diff
@@ -119,14 +119,28 @@ figure.alignnone {
...
+	position: relative;
+}
+
+.single-post .article-body > p:last-child:after,
+.single-post .article-body > ul:last-child > li:last-child:after {
 	background: #a60f2d;
 	content: "";
 	display: inline-block;
-	height: 15px;
+	height: .8em;
```
Comment (slocker3) [inline css/01-layout.css]: 👍

File: css/02-typography.css [hunk 3]
```diff
@@ -372,15 +378,13 @@ main .card-title a:hover {
 }
 
 .card-time {
```
Comment (slocker3) [inline css/02-typography.css]: We should also consider giving this color to `.card-date` as well.

File: style.css (hunk 1 above)
Comment (philcable) [reply style.css]: Good point! I just blindly copied that over from Edit CSS - I'll bump it to 600.

File: css/02-typography.css (hunk 3 above)
Comment (slocker3) [reply css/02-typography.css]: Good question. I think we can hold on this and see what it looks like in all contexts.

**Pull request merged**